e2e/vmss.go

```diff
@@ -223,10 +220,10 @@ func getVmssName(r *mrand.Rand) string {
 	return fmt.Sprintf(vmssNameTemplate, randomLowercaseString(r, 4))
 }
 
-func getBaseVMSSModel(name, sshPublicKey, customData, cseCmd string, opts *scenarioRunOpts) (armcompute.VirtualMachineScaleSet, error) {
+func getBaseVMSSModel(name, sshPublicKey, customData, cseCmd string, opts *scenarioRunOpts) armcompute.VirtualMachineScaleSet {
 	resourceID, err := config.VHDUbuntu1804Gen2Containerd()
 	if err != nil {
-		return armcompute.VirtualMachineScaleSet{}, fmt.Errorf("get resource ID for VHD: %w", err)
+		log.Printf("get resource ID for VHD, will not set default VHD within base VMSS model: %s", err)
```

Comment:
why don't we want to return error here anymore?

Reply:
if a scenario wants to use any VHD other than the default, and it so happens that other VHD does in fact exist, we'd still want the scenario to be runnable

right now, if it's the case that this default VHD doesn't exist, then non of the scenarios will end up being run